src/chat.rs

```diff
@@ -2923,10 +2994,14 @@ pub async fn forward_msgs(context: &Context, msg_ids: &[MsgId], chat_id: ChatId)
             msg.subject = "".to_string();
 
             let new_msg_id: MsgId;
-            if msg.state == MessageState::OutPreparing {
+            if msg.state == MessageState::OutDraft {
+                bail!("cannot forward drafts.");
+            } else if msg.state == MessageState::OutPreparing {
                 let fresh9 = curr_timestamp;
                 curr_timestamp += 1;
-                new_msg_id = chat.prepare_msg_raw(context, &mut msg, fresh9).await?;
+                new_msg_id = chat
+                    .prepare_msg_raw(context, &mut msg, None, fresh9)
```

Comment:
Just a note, these `fresh9`, `fresh10` etc. variables are the result of C to Rust conversion and expansion of sqlite macros, they have no meaning.
Can just pass `current_timestamp` instead and increas it after calling `prepare_msg_raw`.

Reply:
i will change that in this pr, it is a good timing :)